src/components/bottomSheet/bottom-sheet.js

```diff
@@ -85,6 +85,8 @@ function MdBottomSheetDirective($mdBottomSheet) {
  *   of 3.
  *   - `clickOutsideToClose` - `{boolean=}`: Whether the user can click outside the bottom sheet to
  *     close it. Default true.
+ *   - `disableBackdrop` - `{boolean=}`: When set to true, the bottomsheet will not show a backdrop.
+ *     clicking to close the bottomsheet. Default true.
```

Comment:
Thanks, didn't saw that, because I recently changed the attribute name from `useBackdrop` to `disableBackdrop`